earth_enterprise/rpms/build.gradle

```diff
@@ -5,127 +5,313 @@
     which copies build artifacts, binaries and documents in to the /tmp/fusion_os_install path.
  */
 
+import java.nio.file.FileSystems
+import java.nio.file.Files
+import java.nio.file.Path
+import com.netflix.gradle.plugins.utils.JavaNIOUtils
+
 buildscript {
     repositories {
         jcenter()
     }
+
     dependencies {
-        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+        classpath 'com.netflix.nebula:gradle-ospackage-plugin:3.6.0'
```

Comment:
This seems to be what fixes the RPM for Cent OS 6.  The symlinks don't seem to make a differenc.

Reply:
In a way this makes sense to me...I am guessing there is something incompatible in cpio/rpm support changes between what is in older and newer rpms...does this then still work for centos7/rhel7 though?

Reply:
We should probably file a bug against the plugin. You'd expect the newer version to work better :)

Reply:
There, actually, already was such an issue opened for `ospackage`.  I added some comments with the versions that work and the ones that don't work with Cent OS 6.